Code/Mantid/docs/source/interfaces/FilterEventUI.rst

```diff
@@ -0,0 +1,122 @@
+Filter Events Interface
+=======================
+
+.. contents:: Table of Contents
+  :local:
+   
+Description
+-----------
+ 
+The Filter Events interface provides aid to users in order to set up event filter with different 
+:ref:`functionalities-GenerateEventFilter-ref`. This interface utilises the algorithm 
+:ref:`GenerateEventsFilter <algm-GenerateEventsFilter>` which is able to generate event splitters
+according to user’s requirement for filtering events. 
+The generated time splitters are stored either in a SplittersWorkspace or a 
+:ref:`MatrixWorkspace <MatrixWorkspace>`. Both of them will be used by algorithm 
+:ref:`FilterEvents <algm-FilterEvents>` which filters the events from a EventWorkspace to one or 
+multiple EventWorkspaces according to an input `SplittersWorkspace <http://www.mantidproject.org/SplittersWorkspace>`_
+containing a series of splitters. 
+
+On the bottom section the user has an option of :ref:`filterbylogv-GenerateEventFilter-ref`  
+or :ref:`filterbytime-GenerateEventFilter-ref`. User also has an option of filtering events using 
+:ref:`advsetup-FilterEventUI-ref`. 
+
+
+Layout
+------
+
+.. figure:: /images/FilterEventsUI.png
+   :align: center
+   :width: 400
+
+
+Data Loading
+------------
+
+For the data loading step, MantidPlot provides an interface in which the user shall select the 
+appropriate file via *Browse* button in the *File/Run* field. The user must provide a file, 
```

Comment:
The *File/Run* field can also accept an integer as a valid run number.  It works in SNS as long as the instrument name is set up in Mantid.  I am not sure how it works in ISIS.